content/developer/Theme/sass_compiler.adoc

```diff
@@ -0,0 +1,400 @@
+ +
+ +
+
+=== Typical Workflow for Projects
+'''
+* Change a .scss file
+* Commit just the .scss file to git
+* Change a .scss file
+* Commit just the .scss file to git
+* Compile style.scss with the php compiler
+* Commit style.css +
+
+=== Typical Workflow for the Product
+'''
+* Change a .scss file
+* Commit just the .scss file to git
+* Change a .scss file
+* Commit just the .scss file to git
+* Make a pull request. Under the "How To Test" section" add "Rebuild sass". +
+
+
+_Don't commit style.css as we do not want any more bloat added to the product. Style.css will be built during the release process._ +
+ +
+ +
+
+=== Setting Up A Compiler
+'''
+Here are a few guides on how to set up a compiler for Sass. +
+ +
+
+=== Scss PHP Compiler
+'''
+The http://leafo.net/scssphp/[Scss PHP] which used in SuiteP 7.8 or above, ensures that SuiteCRM can compile the Sass code. It also ensures that the style.css uses the same color and measurement formats. +
+
+==== Install Composer
+....
+sudo apt-get install composer
+....
+
+==== Install Scss PHP
+
+* Open a terminal
+* cd /path/to/your/instance
+* Run
+....
+composer require leafo/scssphp
+composer install
+....
+
+==== Using Scss PHP
+
+* Open a terminal
+* cd /path/to/your/instance
+* Run +
+
+*Pre 7.10*
+....
+./vendor/bin/pscss -f compressed themes/SuiteP/css/style.scss >  themes/SuiteP/css/style.css
+....
+
+*7.10 and above*
+....
+./vendor/bin/pscss -f compressed themes/SuiteP/css/Dawn/style.scss >  themes/SuiteP/css/Dawn/style.css
+./vendor/bin/pscss -f compressed themes/SuiteP/css/Day/style.scss >  themes/SuiteP/css/Day/style.css
+./vendor/bin/pscss -f compressed themes/SuiteP/css/Dusk/style.scss >  themes/SuiteP/css/Dusk/style.css
+./vendor/bin/pscss -f compressed themes/SuiteP/css/Night/style.scss >  themes/SuiteP/css/Night/style.css
+....
+
+'''
+
+=== Styling SuiteP
+'''
+The style of each feature of SuiteCRM is broken down into seperate css files. This helps to make the compiler slightly faster and it also helps to make the CSS more organised. The style.scss file is the main stylesheet file. Meaning that it imports all of the other stylesheets into style.css. The variables.scss is the stylesheet that contains all of the colors and measurements which can be easily configured. +
+ +
+
+*https://docs.suitecrm.com/community/contributing-code/coding-standards/[Please ensure that your css complies with our style guide] and http://csslint.net/[please check your code in css lint for issue before committing.]* +
```

Comment:
Don't use bold. 
Please make the links contain less words, they are rendering with strange line breaks this way. It's enough to link the words `style guide` in the first link, and the words `css lint` in the second one.